Unix/doc/allow-deny-handling.md

```diff
@@ -0,0 +1,76 @@
+## Authorization Feature
+
+In April, 2018, the authorization feature was introduced to further control access to OMI.  This is done through two new parameters in omiserver.conf:  **AuthorizedGroups** and **UnauthorizedGroups**. This document describes how these two parameters should be used to limit access to OMI.
```

Comment:
Use `AuthorizedGroups` and `UnauthorizedGroups`. I think back-ticks are more readable, and more readily support cutting/pasting if necessary.

This document was very good, and covered my concerns with no corrections! Just look for all occurrences of `AuthorizedGroups` and `UnauthorizedGroups` and, rather than use bolding, use back-ticks.